src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -320,37 +320,31 @@ partial class BoundStateMachineScope
         protected override OperationKind StatementKind => OperationKind.None;
     }
 
-    partial class BoundLocalDeclaration : IVariableDeclarationStatement
+    partial class BoundLocalDeclaration : IVariableDeclarationStatement, IVariable
```

Comment:
Why this type is implementing `IVariable` again? `IVariable` is an `IOperation`, but with this change, the operation kind of each individual `IVariable` will become `VariableDeclarationStatement` instead of `VariableDeclaration`.